### PR TITLE
tests

### PR DESCRIPTION
**Pull request by vicfriedman**
redirects in test - @jmburges and @AnnJohn

### CODE REVIEW

File: spec/secure_password_spec.rb [hunk 1]
```diff
@@ -42,23 +45,28 @@
...
     it "displays the user's account page if username and password is given" do
+
       user = User.new(username: "avi", password: "I<3Ruby")
       user.save
       post '/login', {"username" => "avi", "password" => "I<3Ruby"}
+      follow_redirect!
       expect(last_response.body).to include('Welcome')
       expect(last_response.body).to include('avi')
-      expect(session[:id]).to not_be(nil)
+      binding.pry
```
Comment (jmburges) [inline spec/secure_password_spec.rb]: yank the binding.pry

**Comment by jmburges** [comment]
one issue then merge it in

**Comment by jmburges** [comment]
also with the pry, this wasn't working. Can you double check this works against the solution?

**Comment by vicfriedman** [comment]
@jmburges removed pry.  test passes on solution branch.

**Pull request merged**